test/e2e/alphagenerate/generate_test.go

```diff
@@ -197,7 +210,24 @@ func ReGenerateProject(kbc *utils.TestContext) {
 	ExpectWithOffset(1, err).NotTo(HaveOccurred())
 	ExpectWithOffset(1, fileContainsExpr).To(BeTrue())
 
-	By("checking if the project file was generated with the expected controller")
+	By("checking if the project file was generated with the expected deploy-image plugin fields")
+	var deployImagePlugin = "deploy-image.go.kubebuilder.io/v1-alpha"
+	fileContainsExpr, err = pluginutil.HasFileContentWith(
+		filepath.Join(kbc.Dir, "testdir2", "PROJECT"), deployImagePlugin)
+	Expect(err).NotTo(HaveOccurred())
+	Expect(fileContainsExpr).To(BeTrue())
+	var deployImagePluginFields = `      kind: Memcached
```

Comment:
```suggestion
	var deployImagePluginFields = `kind: Memcached
```
You can probably drop the spaces here?

Reply:
Good idea.